.bazelrc

```diff
@@ -104,6 +96,27 @@ build:linux --linkopt="-gdwarf-4"
 build:macos --copt="-gdwarf-2"
 build:macos --linkopt="-gdwarf-2"
 
+# Emit full debug info, allowing us to easily analyze core dumps from staging
+# (and, in an emergency, also prod).
+build:debuginfo-full --@rules_rust//:extra_rustc_flag=-Cdebuginfo=2
+build:debuginfo-full --copt=-g2
+
+build:debuginfo-limited --@rules_rust//:extra_rustc_flag=-Cdebuginfo=line-tables-only
+build:debuginfo-limited --copt=-gline-tables-only
+
+#
+# Common Build Configuration
+#
+build --linkopt="-fuse-ld=lld"
+build --@rules_rust//:extra_rustc_flag="-Clink-arg=-fuse-ld=lld"
+build --@rules_rust//:extra_rustc_flag="-Csymbol-mangling-version=v0"
+# We use 64 because it's enough to totally saturate a CI builder so our builds
+# are as fast as possible, and it's less than the default of 256 used with
+# Cargo when incremental compilation is enabled.
+#
+# TODO(parkmycar): Investigate a "super release" build with codegen-units=1
```

Comment:
I remember this being a one-way street in previous companies, which meant that for release builds you'd always have to wait a long time, which made performance benchmarks annoying.

Reply:
Totally fair! Removed this comment